docs/guides/06-aggregations.md

```diff
@@ -0,0 +1,408 @@
+## Aggregating data
+
+In this guide you will explore a set of functions called **aggregations** (aggr.), that will allow you to extract different values from your datasets considering the whole set of features (*global aggr.*), just the ones in the current map extent (*viewport aggr.*) or from spatial aggregations (*cluster aggr.*).
+
+
+### Global aggregations
+When you are working with a dataset, odds are that you are interested in comparing a feature with the rest of them. For example, _"is the population density in this neighborhood bigger than the average?"_. Or maybe you just want to display in a table some basic statistics describing the whole dataset.
+
+For these cases, CARTO VL provides you with a set of functions automatically calculated for the whole dataset (hence *global* functions):
+- `globalMin`: calculates the minimum value.
+- `globalMax`: calculates the maximum value.
+- `globalSum`: calculates the sum of all values.
+- `globalCount`: calculates the number of features.
+- `globalAvg`: calculates the average value.
+- `globalPercentile`: calculates the Nth percentile.
+
+Let's use some of them with our cities dataset. You can start as usual from a [basemap](/developers/carto-vl/examples/maps/guides/getting-started/step-1.html). Take its source code and copy it to a new file called `aggregations.html`. Now you're ready to start.
+
+Then create a visualization adding the cities layer and 3 variables with global functions using this code:
+```js
+const citiesSource = new carto.source.Dataset('populated_places');
+const citiesViz = new carto.Viz(`
+    color: grey
+    width: 10
+    @g_max: globalMax($pop_max)
+    @g_avg: globalAvg($pop_max)
+    @g_p95: globalPercentile($pop_max, 95)
+`);
+const citiesLayer = new carto.Layer('cities', citiesSource, citiesViz);
+citiesLayer.addTo(map);
+```
+> The use of `g_` prefix is not required but it can be useful as an indicator of a _global_ variable. Don't forget to add your credentials as usual, using `carto.setDefaultAuth({ username: 'cartovl', apiKey: 'default_public'});` to be able to connect to the CARTO dataset.
+
+For a review of these variables, add a `console.log` sentence once the layer has loaded:
+```js
+function displayGlobalValues() {
+    const v = citiesViz.variables;
+    console.log(`
+        Maximum: ${v.g_max.value}
+        Average: ${v.g_avg.value.toFixed(0)}
+        95th percentile: ${v.g_p95.value}
+    `);
+}
+citiesLayer.on('loaded', displayGlobalValues);
+```
+> Creating a separated function allows easily deactivating the layer later on with `citiesLayer.off`.
+
+Global functions can be easily combined with other capabilities, such as `filter`. For example, you can display just the biggest cities in the world with this little addition to your `viz`:
```

Comment:
@makella told me yesterday to avoid "easy" and "simple" as much as possible in the guides, because it may be not easy for someone and therefore counterproductive.

Reply:
✨ very good point!